python/tvm/relay/frontend/tflite.py

```diff
@@ -3539,7 +3539,62 @@ def get_tensor_name(subgraph, tensor_idx):
     return subgraph.Tensors(tensor_idx).Name().decode("utf-8")
 
 
-def from_tflite(model, shape_dict, dtype_dict):
+def get_tensor_shape(subgraph, tensor_idx):
+    """Get the tensor shape.
```

Comment:
* minor: the name of the argument doesn't match with the actual argument
* the types are not specified

Please review all docstring being introduced here for those items above.